package.json

```diff
@@ -77,23 +77,34 @@
     "url": "https://github.com/elastic/kibana.git"
   },
   "resolutions": {
+    "**/@mapbox/geojsonhint/minimist": "^1.2.5",
+    "**/@types/angular": "^1.6.56",
+    "**/@types/hapi": "^17.0.18",
+    "**/@types/hoist-non-react-statics": "^3.3.1",
     "**/@types/node": ">=10.17.17 <10.20.0",
     "**/@types/react": "^16.9.19",
     "**/@types/react-router": "^5.1.3",
-    "**/@types/hapi": "^17.0.18",
-    "**/@types/angular": "^1.6.56",
-    "**/@types/hoist-non-react-statics": "^3.3.1",
-    "**/typescript": "3.7.2",
+    "**/deepmerge": "^4.2.2",
+    "**/fast-deep-equal": "^3.1.1",
+    "**/geojson-flatten/minimist": "^1.2.5",
+    "**/geojson-rewind/minimist": "^1.2.5",
```

Comment:
Woah... this seems aggressive and hard to manage... can you explain your logic here a bit?

Reply:
In my first commit I had written it as simply:

```
"**/minimist": "^1.2.5"
```

But I felt there was a smaller risk of something breaking if we split up the forced upgrade between those already using 0.x and those using 1.x. In a perfect world, we could just write it like this:

```
"**/minimist@0.x": "^0.2.1",
"**/minimist@1.x": "^1.2.5"
```

But as far as I can see in the spec there's no way to bind the upgrade to a specific version.

Alternatively, we just revert to the original implementation of upgrading everything to `^1.2.5`. Now that I'm writing this, I'm actually leaning towards that solution again.

Reply:
If the breaking change between v0 and v1 doesn't really impact us then I'm down for going to 1.x across the board too. I assume part of the negatives here are increased warnings because of version mismatches caused by resolve entries... How many packages are still using `minimist@0.x`?

Reply:
5 of our (sub)dependencies are still using v0.x. 6 are using v1.x.

I went over the breaking changes between 0.x and 1.x and I don't think they should cause us any trouble as they, as far as I can see, deal with changes in how the input string is parsed - mostly for special use-cases.

Reply:
5 warnings though, that's a lot of noise. Any idea if we can upgrade something else to get those updated? Or at least to reduce the number of warnings to 1 or 2?

Reply:
I've tried to update all packages that could be normally updated in separate PRs. I've just updated this PR with what's left. These I wouldn't hold my breath for, so I'm using the package resolutions feature to forcefully upgrade those.